### PR TITLE
Translated CMS 2020 to Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
Makes progress on #1750

### CODE REVIEW

File: src/content/nl/2020/cms.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+
+Een "goed" LCP wordt beschouwd als minder dan 2,5 seconden. De gemiddelde website op een van de vijf beste CMS'en heeft geen goed LCP. Alleen Drupal op desktop scoort hier meer dan 50%. We zien grote verschillen tussen desktop- en mobiele scores: WordPress scoort redelijk zelfs 33% op desktop en 25% op mobiel, maar Squarespace scoort 37% op desktop en slechts 12% op mobiel.
+
+{{ figure_markup(
+  image="cms-real-user-largest-contentful-paint-experiences.png",
+  caption="Largest Contentful Paint-ervaringen door echte gebruikers.",
+  description='Staafdiagram met de top 5 CMS'en en of ze een "goede" Largest Contentful Paint-ervaring hebben. WordPress is middelmatig met 33% op desktop en 25% op mobiel, Drupal is de beste met 61% op desktop en 47% op mobiel, Joomla is de tweede beste met 48% op desktop en 28% op mobiel, Squarespace heeft 37% op desktop maar slechts 12% op mobiel en Wix is het laagste met 9% op desktop en 9% op mobiel.',
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:354]:
```suggestion
  description="Staafdiagram met de top 5 CMS'en en of ze een \"goede\" Largest Contentful Paint-ervaring hebben. WordPress is middelmatig met 33% op desktop en 25% op mobiel, Drupal is de beste met 61% op desktop en 47% op mobiel, Joomla is de tweede beste met 48% op desktop en 28% op mobiel, Squarespace heeft 37% op desktop maar slechts 12% op mobiel en Wix is het laagste met 9% op desktop en 9% op mobiel.",
```

File: src/content/nl/2020/cms.md [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+
+FID is erg snel voor de gemiddelde CMS-website op desktop - alleen Wix scoort lager dan 100% - en gemengd op mobiel. De meeste CMS'en leveren mobiele FID op een gemiddelde site binnen een redelijk bereik van de desktopscore. Voor Wix is het aantal websites met een goede FID op mobiel bijna de helft van het desktop-totaal.
+
+{{ figure_markup(
+  image="cms-real-user-first-input-delay-experiences.png",
+  caption="First Input Delay-ervaringen door echte gebruikers.",
+  description='Staafdiagram met de top 5 CMS'en en of ze een "goede" ervaring met de eerste invoervertraging hebben. Ze hebben allemaal een ervaringsscore van 100% op desktop, behalve Wix met 87%. Voor mobiel heeft WordPress 88%, Drupal 76%, Joomla 71%, Squarespace 91% en Wix 46%.',
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:372]:
```suggestion
  description="Staafdiagram met de top 5 CMS'en en of ze een \"goede\" ervaring met de eerste invoervertraging hebben. Ze hebben allemaal een ervaringsscore van 100% op desktop, behalve Wix met 87%. Voor mobiel heeft WordPress 88%, Drupal 76%, Joomla 71%, Squarespace 91% en Wix 46%.",
```

File: src/content/nl/2020/cms.md [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+#### Cumulative Layout Shift
+
+Cumulatieve Layout Shift (CLS) meet de instabiliteit van inhoud op een webpagina na de eerste 500 ms gebruikersinvoer en na de eerste gebruikersinvoer. Dit is met name belangrijk op mobiele apparaten, waar de gebruiker zal tikken waar hij een actie wil ondernemen, zoals een zoekbalk, alleen om de locatie te verplaatsen als aanvullende afbeeldingen, advertenties of soortgelijke ladingen.
+
+Een score van 0,1 of lager wordt gemeten als "goed", meer dan 0,25 is "slecht" en alles daartussenin is "moet worden verbeterd".
+
+{{ figure_markup(
+  image="cms-real-user-cumulative-layout-shift-experiences.png",
+  caption="Cumulatieve Layout Shift-ervaringen door echte gebruikers.",
+  description='Staafdiagram met de top 5 CMS'en en of ze een "goede" ervaring hebben met Cumulative Layout Shift. WordPress heeft 47% van de desktopsites met een "goede ervaring" en 57% van de mobiele sites. Drupal heeft 58% voor desktop en 70% voor mobiel, Joomla heeft 51% voor desktop en 63% voor mobiel, Squarespace heeft 35% voor desktop en 44% voor mobiel, en Wix heeft 58% voor desktop en 59% voor mobiel.',
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:392]:
```suggestion
  description="Staafdiagram met de top 5 CMS'en en of ze een \"goede\" ervaring hebben met Cumulative Layout Shift. WordPress heeft 47% van de desktopsites met een \"goede ervaring\" en 57% van de mobiele sites. Drupal heeft 58% voor desktop en 70% voor mobiel, Joomla heeft 51% voor desktop en 63% voor mobiel, Squarespace heeft 35% voor desktop en 44% voor mobiel, en Wix heeft 58% voor desktop en 59% voor mobiel.",
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Man you are on fire!!!

As you use "CMS'en" can't switch to single quotes so need to escape instead.

File: src/content/nl/2020/cms.md [hunk 4]
```diff
@@ -0,0 +1,527 @@
+---
+part_number: III
+chapter_number: 15
+title: CMS
+description: CMS-hoofdstuk van de Web Almanac van 2020 over CMS-acceptatie, hoe CMS-suites worden gebouwd, gebruikerservaring van CMS-aangedreven websites en CMS-innovatie.
+authors: [alexdenning]
+reviewers: [sirjonathan, ernee, amedina]
+analysts: [GregBrimble, rviscomi]
+translators: [noah-vdv]
+alexdenning_bio: Alex Denning is de oprichter van <a href="https://getellipsis.com/">Ellipsis Marketing</a>, een marketingbureau voor WordPress-bedrijven. Alex is een WordPress Core-bijdrager en heeft geholpen bij het organiseren van <a href="https://london.wordcamp.org/">WordCamp London</a>.
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:10]: In #1781 and #1793 we added the ability to highlight a light as being in a different language to the text by adding the `hreflang` attribute to links.

```suggestion
alexdenning_bio: Alex Denning is de oprichter van <a hreflang="en" href="https://getellipsis.com/">Ellipsis Marketing</a>, een marketingbureau voor WordPress-bedrijven. Alex is een WordPress Core-bijdrager en heeft geholpen bij het organiseren van <a hreflang="en" href="https://london.wordcamp.org/">WordCamp London</a>.
```

This then looks like this:

image

Could you try to add these attributes to non-Dutch links from now on? Note likes including "en" like MDN and wikipedia links will automatically get this added so no need to do that for them.

File: src/content/nl/2020/cms.md [hunk 5]
```diff
@@ -0,0 +1,527 @@
...
+
+Er is een grote variatie in het type en de omvang van de ondersteunende CMS'en voor bouwterreinen; sommige bieden kant-en-klare sjablonen die worden aangevuld met gebruikersinhoud, en andere vereisen veel meer gebruikersbetrokkenheid voor het ontwerpen en bouwen van de sitestructuur.
+
+Als we aan CMS'en denken, moeten we rekening houden met alle componenten die een rol spelen bij de levensvatbaarheid van een dergelijk systeem om een platform te bieden voor het publiceren van inhoud op internet. Al deze componenten vormen een ecosysteem rond het CMS-platform en omvatten hostingproviders, extensieontwikkelaars, ontwikkelingsbureaus, sitebouwers, enz. Dus als we het hebben over een CMS, verwijzen we meestal naar zowel het platform zelf als het omliggende ecosysteem. .
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:31]:
```suggestion
Als we aan CMS'en denken, moeten we rekening houden met alle componenten die een rol spelen bij de levensvatbaarheid van een dergelijk systeem om een platform te bieden voor het publiceren van inhoud op internet. Al deze componenten vormen een ecosysteem rond het CMS-platform en omvatten hostingproviders, extensieontwikkelaars, ontwikkelingsbureaus, sitebouwers, enz. Dus als we het hebben over een CMS, verwijzen we meestal naar zowel het platform zelf als het omliggende ecosysteem.
```

File: src/content/nl/2020/cms.md [hunk 6]
```diff
@@ -0,0 +1,527 @@
...
+        <td class="numeric">5,43%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="CMS-adoptiestatistieken.", sheets_gid="1638210080", sql_file="cms_adoption_compared_to_2019.sql") }}</figcaption>
+</figure>
+
+De toename van desktopwebpagina's aangedreven door een CMS-platform is 5,43% ten opzichte van vorig jaar. Op mobiel is deze stijging ongeveer een kwart hoger, namelijk 6,71%.
+
+Net als bij [vorig jaar](../2019/cms#cms-adoption), zien we andere resultaten van andere datasets voor het bijhouden van marktaandeel van CMS-platforms, zoals [W3Techs](https://w3techs.com/technologies/history_overview/content_management). W3Techs meldt op het moment van schrijven dat 60,6% van de webpagina's wordt gemaakt door CMS'en, tegen 56,4% een jaar geleden. Dit is een stijging van 6,4%, wat in grote lijnen overeenkomt met onze bevindingen.
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:93]: That `hreflang` functionality does mean you have to convert markdown links to HTML links if you want to use them which is a bit of extra effort 😢 (we probably should change all the English ones to make this easier for Translators):

```suggestion
Net als bij [vorig jaar](../2019/cms#cms-adoption), zien we andere resultaten van andere datasets voor het bijhouden van marktaandeel van CMS-platforms, zoals <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management">W3Techs</a>. W3Techs meldt op het moment van schrijven dat 60,6% van de webpagina's wordt gemaakt door CMS'en, tegen 56,4% een jaar geleden. Dit is een stijging van 6,4%, wat in grote lijnen overeenkomt met onze bevindingen.
```

File: src/content/nl/2020/cms.md [hunk 7]
```diff
@@ -0,0 +1,527 @@
...
+  sheets_gid="1594044364",
+  sql_file="top_cmss_yoy_all_clients.sql"
+  )
+}}
+
+### WordPress gebruik
+
+WordPress domineert deze ruimte en verdient daarom verdere discussie.
+
+[WordPress is een open source project](https://wordpress.org/about/) met als missie "publiceren democratiseren". Het CMS is gratis. Hoewel dit waarschijnlijk een belangrijke factor is in het adoptiegedeelte, zijn de twee op één na populairste CMS'en - Joomla en Drupal - ook gratis. De WordPress-gemeenschap, bijdragers en het zakelijke ecosysteem zijn waarschijnlijk de belangrijkste onderscheidende factoren.
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:303]:
```suggestion
[WordPress is een open source project](https://nl.wordpress.org/about/) met als missie "publiceren democratiseren". Het CMS is gratis. Hoewel dit waarschijnlijk een belangrijke factor is in het adoptiegedeelte, zijn de twee op één na populairste CMS'en - Joomla en Drupal - ook gratis. De WordPress-gemeenschap, bijdragers en het zakelijke ecosysteem zijn waarschijnlijk de belangrijkste onderscheidende factoren.
```

File: src/content/nl/2020/cms.md [hunk 8]
```diff
@@ -0,0 +1,527 @@
...
+
+Nu het adoptie-aandeel van CMS'en is onderzocht, gaan we nu onze aandacht richten op de gebruikerservaring.
+
+## CMS-gebruikerservaring
+
+CMS'en moeten een goede gebruikerservaring bieden. Omdat een groot deel van het web afhankelijk is van CMS'en om pagina's te bedienen, is het de verantwoordelijkheid van het CMS op platformniveau om ervoor te zorgen dat de gebruikerservaring goed is. Ons doel is om licht te werpen op de echte gebruikerservaring bij het gebruik van CMS-aangedreven websites.
+
+Om dit te bereiken, richten we onze analyse op een aantal door de gebruiker waargenomen prestatiestatistieken, die zijn vastgelegd in de drie Core Web Vitals-statistieken, evenals de Lighthouse-scores in de categorieën SEO en Toegankelijkheid.
+
+### Chrome-gebruikerservaringrapport
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:333]: I would leave this in English as it's a brand name:

```suggestion
### Chrome User Experience Report
```

File: src/content/nl/2020/cms.md [hunk 9]
```diff
@@ -0,0 +1,527 @@
...
+  sheets_gid="1671385936",
+  sql_file="median_lighthouse_score.sql"
+  )
+}}
+
+Alle top 5 CMS'en scoren hier hoog met mediane scores van 0,83 of hoger, sommige zelfs tot 0,93. SEO kan ervan afhangen dat de website-eigenaar gebruik maakt van de mogelijkheden van een CMS, maar als deze opties gemakkelijk te gebruiken zijn in een CMS, en goede standaardinstellingen, kan dit grote voordelen hebben voor sites die op die CMS'en draaien.
+
+#### Toegankelijkheid
+
+Een toegankelijke website is een site die zo is ontworpen en ontwikkeld dat mensen met een handicap deze kunnen gebruiken. Webtoegankelijkheid komt ook ten goede aan mensen zonder handicap, zoals mensen met langzame internetverbindingen. [Een volledige discussie is hier te zien](https://www.w3.org/WAI/fundamentals/accessibility-intro/#what), en in ons hoofdstuk [Toegankelijkheid](./accessibility).
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:423]:
```suggestion
Een toegankelijke website is een site die zo is ontworpen en ontwikkeld dat mensen met een handicap deze kunnen gebruiken. Webtoegankelijkheid komt ook ten goede aan mensen zonder handicap, zoals mensen met langzame internetverbindingen. <a hreflang="en" href="https://www.w3.org/WAI/fundamentals/accessibility-intro/#what">Een volledige discussie is hier te zien</a>, en in ons hoofdstuk [Toegankelijkheid](./accessibility).
```

File: src/content/nl/2020/cms.md [hunk 10]
```diff
@@ -0,0 +1,527 @@
...
+
+Lighthouse biedt een reeks toegankelijkheidscontroles en retourneert een gewogen gemiddelde van alle audits (zie [Scoring Details](https://web.dev/accessibility-scoring/) voor een volledige lijst van hoe elke audit wordt gewogen).
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:425]:
```suggestion
Lighthouse biedt een reeks toegankelijkheidscontroles en retourneert een gewogen gemiddelde van alle audits (zie <a hreflang="en" lang="en" https://web.dev/accessibility-scoring/">Scoring Details</a> voor een volledige lijst van hoe elke audit wordt gewogen).
```

File: src/content/nl/2020/cms.md [hunk 11]
```diff
@@ -0,0 +1,527 @@
...
+
+De lat zou hier echter hoger moeten zijn. Een gemiddelde score van 0,78 voor alle CMS-en laat nog aanzienlijke ruimte voor verbetering, en de maximale score van 0,98 laat zien dat zelfs het "beste" CMS voor naleving van toegankelijkheid voor verbetering vatbaar is. Het verbeteren van de toegankelijkheid is essentieel en urgent.
+
+## Milieu-impact
+
+Dit jaar hebben we geprobeerd de impact van CMS'en op het milieu beter te begrijpen. [De informatie- en communicatietechnologie (ICT) -industrie is verantwoordelijk voor 2% van de wereldwijde koolstofemissies](https://www.nature.com/articles/d41586-018-06610-y), en datacentra zijn specifiek verantwoordelijk voor 0,3% van wereldwijde koolstofemissies. Hiermee komt de CO2-voetafdruk van de ICT-industrie overeen met de uitstoot van brandstof door de luchtvaartindustrie. We hebben hier geen gegevens over de rol van CMS'en, maar uit ons onderzoek blijkt dat 42% van de websites een CMS gebruikt, het is duidelijk dat CMS'en een belangrijke rol spelen bij de efficiëntie van websites en hun impact op het milieu.
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:445]:
```suggestion
Dit jaar hebben we geprobeerd de impact van CMS'en op het milieu beter te begrijpen. <a hreflang="en" href="https://www.nature.com/articles/d41586-018-06610-y">De informatie- en communicatietechnologie (ICT) -industrie is verantwoordelijk voor 2% van de wereldwijde koolstofemissies</a>, en datacentra zijn specifiek verantwoordelijk voor 0,3% van wereldwijde koolstofemissies. Hiermee komt de CO2-voetafdruk van de ICT-industrie overeen met de uitstoot van brandstof door de luchtvaartindustrie. We hebben hier geen gegevens over de rol van CMS'en, maar uit ons onderzoek blijkt dat 42% van de websites een CMS gebruikt, het is duidelijk dat CMS'en een belangrijke rol spelen bij de efficiëntie van websites en hun impact op het milieu.
```

File: src/content/nl/2020/cms.md [hunk 12]
```diff
@@ -0,0 +1,527 @@
...
+
+In ons onderzoek is gekeken naar het gemiddelde CMS-paginagewicht in KB en dit in kaart gebracht op de CO2-uitstoot met behulp van logica van [carbonapi](https://gitlab.com/wholegrain/carbon-api-2-0/-/blob/master/include/carbonapi.php#L342). Dit leverde de volgende resultaten op, uitgesplitst naar desktop en mobiel:
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:447]:
```suggestion
In ons onderzoek is gekeken naar het gemiddelde CMS-paginagewicht in KB en dit in kaart gebracht op de CO2-uitstoot met behulp van logica van <a hreflang="en" href="https://gitlab.com/wholegrain/carbon-api-2-0/-/blob/master/include/carbonapi.php#L342">carbonapi</a>. Dit leverde de volgende resultaten op, uitgesplitst naar desktop en mobiel:
```

File: src/content/nl/2020/cms.md [hunk 13]
```diff
@@ -0,0 +1,527 @@
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=74536341&format=interactive",
+  sheets_gid="1846606749",
+  sql_file="third_party_bytes_and_requests_on_cmss.sql"
+  )
+}}
+
+Video draagt hier een groter percentage per brontype bij. Het efficiënter maken van video of andere mechanismen, zoals de impact van het stoppen van automatisch afspelen, zijn interessante gebieden voor toekomstig onderzoek. Hier ligt onze focus op afbeeldingen. Populaire afbeeldingsindelingen zijn JPEG, PNG, GIF, SVG, WebP en ICO. Hiervan is [WebP is het meest efficiënt in de meeste situaties](https://developers.google.com/speed/webp/), met WebP lossless afbeeldingen [26% kleiner] (https://developers.google.com/speed/webp/docs/webp_lossless_alpha_study#resultaten) dan vergelijkbare PNG's en [25-34% kleiner](https://developers.google.com/speed/webp/docs/webp_study) dan vergelijkbare JPG's. We zien echter dat WebP het op één na minst populaire afbeeldingsformaat is op alle CMS-pagina's:
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:501]:
```suggestion
Video draagt hier een groter percentage per brontype bij. Het efficiënter maken van video of andere mechanismen, zoals de impact van het stoppen van automatisch afspelen, zijn interessante gebieden voor toekomstig onderzoek. Hier ligt onze focus op afbeeldingen. Populaire afbeeldingsindelingen zijn JPEG, PNG, GIF, SVG, WebP en ICO. Hiervan is <a hreflang="en" href="https://developers.google.com/speed/webp/">WebP is het meest efficiënt in de meeste situaties</a>, met WebP lossless afbeeldingen <a hreflang="en" href="https://developers.google.com/speed/webp/docs/webp_lossless_alpha_study#results">26% kleiner</a> dan vergelijkbare PNG's en <a hreflang="en" href="https://developers.google.com/speed/webp/docs/webp_study">25-34% kleiner</a> dan vergelijkbare JPG's. We zien echter dat WebP het op één na minst populaire afbeeldingsformaat is op alle CMS-pagina's:
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Top quality translation again (well from a technical point of view - can't talk too much about the Dutch!)

Spotted a few things and then can merge this. BTW if you go into the Files tab you can batch accept any of these which is quicker than accepting one by one.

**Comment by noah-vdv** [comment]
So basically for every english link we add an `hreflang="en"` except for links with "en" in URL?

**Comment by bazzadp** [comment]
> So basically for every english link we add an `hreflang="en"` except for links with "en" in URL?

Ideally yes, except for internal links (as they should eventually be in Dutch), links that we have already converted to Dutch (e.g. I didn't add it to the Wordpress one as instead changed it to the Dutch version o that page), and links with `/en/` or `/en-US`, `https://en.` in them.

File: src/content/nl/2020/cms.md [hunk 14]
```diff
@@ -0,0 +1,527 @@
...
+  )
+}}
+
+Alle top 5 CMS'en scoren hier hoog met mediane scores van 0,83 of hoger, sommige zelfs tot 0,93. SEO kan ervan afhangen dat de website-eigenaar gebruik maakt van de mogelijkheden van een CMS, maar als deze opties gemakkelijk te gebruiken zijn in een CMS, en goede standaardinstellingen, kan dit grote voordelen hebben voor sites die op die CMS'en draaien.
+
+#### Toegankelijkheid
+
+Een toegankelijke website is een site die zo is ontworpen en ontwikkeld dat mensen met een handicap deze kunnen gebruiken. Webtoegankelijkheid komt ook ten goede aan mensen zonder handicap, zoals mensen met langzame internetverbindingen. <a hreflang="en" href="https://www.w3.org/WAI/fundamentals/accessibility-intro/#what">Een volledige discussie is hier te zien</a>, en in ons hoofdstuk [Toegankelijkheid](./accessibility).
+
+Lighthouse biedt een reeks toegankelijkheidscontroles en retourneert een gewogen gemiddelde van alle audits (zie <a hreflang="en" lang="en" https://web.dev/accessibility-scoring/">Scoring Details</a> voor een volledige lijst van hoe elke audit wordt gewogen).
```
Comment (bazzadp) [inline src/content/nl/2020/cms.md:425]:
```suggestion
Lighthouse biedt een reeks toegankelijkheidscontroles en retourneert een gewogen gemiddelde van alle audits (zie <a hreflang="en" lang="en" href="https://web.dev/accessibility-scoring/">Scoring Details</a> voor een volledige lijst van hoe elke audit wordt gewogen).
```

**Review by bazzadp — APPROVED** [review]

**Pull request merged**